capstone-frontend/src/components/MyClubs.js

```diff
@@ -34,13 +25,15 @@ class MyClubs extends Component {
 
   render() {
     let clubArray = [];
-    this.state.clubs.forEach(c => {
+    [...this.state.clubs].forEach(c => {
```

Comment:
Was there a specific reason you wanted to use the spread operator here vs. the original code?

Reply:
At some point I was getting an error without it, but it seems that was related to the backend response. Changing back to the ```.forEach()``` now. Thanks!